### PR TITLE
fix: turbo codemode

### PR DESCRIPTION
**Pull request by emersonlaurentino**
After #2156 here are some codemode fixes that need to be done.

ref: https://turbo.build/repo/docs/reference/codemods

### CODE REVIEW

File: turbo.json [hunk 1]
```diff
@@ -28,25 +28,44 @@
...
     "dev": {
       "cache": false
     },
     "dev:server": {
       "cache": false
     },
     "@faststore/api#dev:graphql": {
+      "outputs": [
+        "dist/**",
+        "build/**"
```
Comment (icazevedo) [inline turbo.json:55]: It doesn't output to build, neither does `faststore/api#build:cjs` or `@faststore/api#build:esm`

Comment (emersonlaurentino) [reply turbo.json:55]: this was automatically generated by codemode.

But I think it's correct, don't you? doesn't dev create these folders?

Comment (icazevedo) [reply turbo.json:55]: No, it only creates `dist` 🤔

Comment (emersonlaurentino) [reply turbo.json:55]: I'll check it

Comment (emersonlaurentino) [reply turbo.json]: done, @icazevedo can u review again?

**Pull request merged**